dev/Detours/README.md

```diff
@@ -0,0 +1,6 @@
+This is a fork of Detours from https://github.com/microsoft/Detours
```

Comment:
is the normal way to consume this version of Detours?

Reply:
Detours has an open issue on the subject
[Detours via Nuget package ?](https://github.com/microsoft/Detours/issues/124)

There's also [this](https://github.com/microsoft/Detours/issues/163) so we'll need to re-fork (in a future PR) if the Nuget / whatever gap isn't addressed soon